acceptancetests/utility.py

```diff
@@ -420,3 +423,46 @@ def assert_dict_is_subset(sub_dict, super_dict):
         raise JujuAssertionError(
             'Found: {} \nExpected: {}'.format(super_dict, sub_dict))
     return True
+
+def add_model(client):
```

Comment:
You also need to change existing usages of add_model and current_model to use these new helpers. I think the current usages are in assess_destroy_model.py, but do a check